src/events/event.helper.ts

```diff
@@ -255,7 +246,7 @@ export class CalEventHelper {
         datestring: string,
         file: { path: string; basename: string }
     ): ParseDate {
-        let datebits = datestring.split(/(?<!^)-/);
+        let datebits = datestring.split(/(?<!^)[-–—]/);
```

Comment:
This is a bit dicey. iOS webkit < v16.4 didn't support regex lookbehinds, so the plugin will fail to install on anyone running that version of iOS

We could probably assume that is a very low number...

Reply:
the look-behind was already present? (longstanding)
I'm not sure we've hit a problem with installation of calendarium..